src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2243,14 +2243,26 @@ private void Insert(object parameters)
         }
 
         internal void OpenOnboardingGuideFile()
-        {  
+        {
             var jsonDynFile = ResourceUtilities.LoadContentFromResources(GuidesManager.OnboardingGuideWorkspaceEmbeededResource, Assembly.GetExecutingAssembly(), false, false);
             OpenFromJson(new Tuple<string, bool>(jsonDynFile, true));
         }
 
         private bool CanOpen(object parameters)
         {
             var filePath = parameters as string;
+
+            if (filePath.Contains(".zip\\") || filePath.Contains(".ZIP\\"))
```

Comment:
@LongNguyenP I think we need something more concrete as in Windows a directory can also be named with `.zip` in its name like, `ABCD.zip`. And in that case this check will cause problems. May be incrementally check if each part of the path is a directory except the last one?

Reply:
what about 7zip archives? (.7z)